pam/integration-tests/integration_test.go

```diff
@@ -239,6 +256,22 @@ func saveArtifactsForDebug(t *testing.T, artifacts []string) {
 	}
 }
 
+func prepareCLILogging(t *testing.T) string {
+	t.Helper()
+
+	cliLog := filepath.Join(t.TempDir(), "authd-pam-cli.log")
+	t.Cleanup(func() {
+		out, err := os.ReadFile(cliLog)
+		if os.IsNotExist(err) {
+			return
+		}
+		require.NoError(t, err)
+		t.Log(string(out))
+	})
+
+	return cliLog
+}
+
```

Comment:
I think we don't care about this now that we don't assert on the logs anymore, right?

Reply:
Well this was part of #233 and while we don't care per se it's quite useful in case of failures since it gives us the cli logs to inspect.

So maybe still worth to have it in case we have unexpected failures in launchpad builds?

Reply:
You are saving the logs to a temporary dir that gets removed when the test ends, so we don't get anything at all. We could add this logic to the SaveArtifactsForDebug function that saves the artefacts on a temporary dir that is not related to the tests themselves, but it will require more changes and it's not a priority right now. So I think removing this the best option for now.

Reply:
Indeed the artifact was added to the artifact dir, plus as you can see in the cleanup func we do `t.Log(string(out))`, so it's logged (try this branch with `-v`) and on failure is shown.

So changes are all there already (unless I forgot to pick something), but still let me know if you prefer to keep this for later.

Reply:
Indeed as per https://github.com/ubuntu/authd/pull/226/commits/dd51fdfbec77e0f932c313918435a9906d0aa644 it should do it all

Reply:
I think from the disussion this should be separated, no?